project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.block.inc

```diff
@@ -24,7 +24,7 @@ function _c4m_content_group_preprocess_header_name_banner(array &$variables) {
 }
 
 /**
- * The group name.
+ * Og group title redesign.
```

Comment:
@anvmn 
Please elaborate the docs since this function now handles both, _group_ and _project_ types.